api/hasura/actions/logout_user.ts

```diff
@@ -0,0 +1,49 @@
+import type { VercelRequest, VercelResponse } from '@vercel/node';
+import { z } from 'zod';
+
+import { gql } from '../../../api-lib/Gql';
+import { verifyHasuraRequestMiddleware } from '../../../api-lib/validate';
+import { GraphQLError } from '../../../src/lib/gql/zeusHasuraAdmin';
+import { HasuraUserSessionVariables } from '../../../src/lib/zod';
+
+async function handler(request: VercelRequest, response: VercelResponse) {
+  try {
+    const { hasuraProfileId } = HasuraUserSessionVariables.parse(
+      request.body.session_variables
+    );
+    await gql.q('mutation')({
+      delete_personal_access_tokens: [
+        {
+          where: {
+            tokenable_id: { _eq: hasuraProfileId },
+            tokenable_type: { _eq: 'App\\Models\\Profile' },
+          },
+        },
+        {
+          affected_rows: true,
+        },
+      ],
+    });
+    return response.status(200).json({ id: hasuraProfileId });
```

Comment:
in the future some of these type of calls might be better served by a 204 no content status, but i'm not sure how hasura handles that

Reply:
i think hasura accepts any 2xx status